pkg/sif/descriptor_input.go

```diff
@@ -253,6 +253,30 @@ func OptSBOMMetadata(f SBOMFormat) DescriptorInputOpt {
 	}
 }
 
+var errMetadataTooLarge = errors.New("metadata value too large")
```

Comment:
I think it's actually the _media type_ that's too long here?

```suggestion
var errMediaTypeTooLarge = errors.New("media type too large")
```